blackfootwords/models.py

```diff
@@ -26,10 +26,13 @@ class Variety(CustomModelMixin, common.Language):
     glottocode = Column(Unicode)
 
 
+@implementer(interfaces.IValue)
+class Lemma(CustomModelMixin, common.Value):
+    pk = Column(Integer, ForeignKey('value.pk'), primary_key=True)
+    categories = Column(Unicode)
```

Comment:
We also have to add the column to the clld app database. Again, this is done by overriding the default table specification - inheriting from the default `common.Value`.
In the database, this will create an additional table `lemma`, joined to the default table with rows having the same primary key. The technique we use here is called [joined table inheritance](https://docs.sqlalchemy.org/en/14/orm/inheritance.html#joined-table-inheritance).